app/helpers/donations_helper.rb

```diff
@@ -4,7 +4,10 @@ def global_donations
     Donation.all.sum(:amount)
   end
 
-  def pretty_dollars(value)
-    "$" + number_with_delimiter(value.to_i)
+  def pretty_dollars(value, decimals=0)
+  	decimal_amount = '%0.' + decimals.to_s + 'f'
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Tab detected.